### PR TITLE
[improve](catalog recycle bin) show data size info when show catalog recycle bin

### PR DESCRIPTION
**Pull request by mymeiyi**
## Proposed changes

add `DataSize` and `RemoteDataSize` info when `show catalog recycle bin`
before:
```
mysql> show catalog recycle bin;
+-----------+------------------------------------+--------+---------+-------------+---------------------+
| Type      | Name                               | DbId   | TableId | PartitionId | DropTime            |
+-----------+------------------------------------+--------+---------+-------------+---------------------+
| Database  | db2                                | 152078 |         |             | 2024-01-30 19:01:27 |
| Database  | regression_test_insert_p0          | 11040  |         |             | 2024-01-30 17:16:10 |
| Table     | datatype                           | 11040  | 51211   |             | 2024-01-30 16:56:13 |
| Table     | datatype                           | 11040  | 150090  |             | 2024-01-30 17:16:10 |
| Table     | insert_group_commit_into_duplicate | 11040  | 51283   |             | 2024-01-30 16:56:14 |
| Partition | p_20230126                         | 13009  | 138136  | 138130      | 2024-01-29 19:02:51 |
+-----------+------------------------------------+--------+---------+-------------+---------------------+
```

after:
```
mysql> show catalog recycle bin;
+-----------+------------------------------------+--------+---------+-------------+---------------------+------------+----------------+
| Type      | Name                               | DbId   | TableId | PartitionId | DropTime            | DataSize   | RemoteDataSize |
+-----------+------------------------------------+--------+---------+-------------+---------------------+------------+----------------+
| Database  | db2                                | 152078 |         |             | 2024-01-30 19:01:27 | 0.000      | 0.000          |
| Database  | regression_test_insert_p0          | 11040  |         |             | 2024-01-30 17:16:10 | 245.977 KB | 0.000          |
| Table     | datatype                           | 11040  | 51211   |             | 2024-01-30 16:56:13 | 2.139 KB   | 0.000          |
| Table     | datatype                           | 11040  | 150090  |             | 2024-01-30 17:16:10 | 2.139 KB   | 0.000          |
| Table     | insert_group_commit_into_duplicate | 11040  | 51283   |             | 2024-01-30 16:56:14 | 2.479 KB   | 0.000          |
| Partition | p_20230126                         | 13009  | 138136  | 138130      | 2024-01-29 19:02:51 | 0.000      | 0.000          |
+-----------+------------------------------------+--------+---------+-------------+---------------------+------------+----------------+
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mymeiyi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/CatalogRecycleBin.java [hunk 1]
```diff
@@ -980,30 +983,7 @@ protected void runAfterCatalogReady() {
     }
 
     public List<List<String>> getInfo() {
```
Comment (SWJTU-ZhangLei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/CatalogRecycleBin.java:985]: Does this method has race condition with recyclebin thread?

**Comment by mymeiyi** [comment]
run pipelinex_p0

**Comment by gavinchou** [comment]
LGTM

**Review by yujun777 — APPROVED** [review]
LGTM

**Comment by mymeiyi** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**